keel-clouddriver/src/test/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCacheTest.kt

```diff
@@ -0,0 +1,119 @@
+package com.netflix.spinnaker.keel.clouddriver
+
+import com.natpryce.hamkrest.isA
+import com.natpryce.hamkrest.isEmpty
+import com.natpryce.hamkrest.should.shouldMatch
+import com.netflix.spinnaker.hamkrest.shouldEqual
+import com.netflix.spinnaker.hamkrest.shouldThrow
+import com.netflix.spinnaker.keel.clouddriver.model.Moniker
+import com.netflix.spinnaker.keel.clouddriver.model.Network
+import com.netflix.spinnaker.keel.clouddriver.model.SecurityGroup
+import com.netflix.spinnaker.keel.clouddriver.model.Subnet
+import com.nhaarman.mockito_kotlin.doReturn
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.whenever
+import org.junit.jupiter.api.Test
+
+object MemoryCloudDriverCacheTest {
```

Comment:
Yeah, ok. JUnit5 greatly superior than Spock. Will likely make that the default for things going forward... actually, will probably PR a refactor of all tests before it gets out of hand.

Reply:
Yeah, let's do that.